generator/ServiceClientGeneratorLib/GeneratorDriver.cs

```diff
@@ -1470,6 +1470,9 @@ public static void RemoveLegacyFiles(string sdkRootFolder)
             var legacyFolderNames = new HashSet<string>
             {
                 "_bcl35",
+                Utils.PathCombineAlt("Generated", "_bcl45"),
```

Comment:
Added these so that we also delete folders for the "test services" (e.g. https://github.com/aws/aws-sdk-net/tree/main/sdk/test/Services/PaginatorsTest/Generated/Model/_bcl45%2Bnetstandard)